packages/@aws-cdk/aws-logs-destinations/test/integ.kinesisfirehose-to-s3.ts

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+import * as iam from '@aws-cdk/aws-iam';
+import * as firehose from '@aws-cdk/aws-kinesisfirehose';
+import * as logs from '@aws-cdk/aws-logs';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as cdk from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as dests from '../lib';
+
```

Comment:
Please add some steps for verifying the resources were wired up correctly (see: [CONTRIBUTING#integration-tests](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#integration-tests)). Something like:
- `aws firehose put-record ...`
- `aws get-log-record ...`

Reply:
Done, thanks